src/js/scripts.js

```diff
@@ -0,0 +1,7 @@
+global.jQuery = require('jquery');
```

Comment:
Could you explain why you decided to use JQuery on this project?

Reply:
jQuery was used in order to create the jQuery plugin $().crazeCarousel. 

In that component, jQuery's $ selector is used many times to find HTML elements to make the code more readable (instead of using document.getElementByClassName()), but the most important use was jQuery's **animate()** function to create the sliding of the carousel.